src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -306,3 +315,227 @@ def test_WINNF_FT_S_MCP_1(self, config_filename):
        to 3700 MHz to protect configurable IAP-protected entities and DPAs
     """
     config = loadConfig(config_filename)
+
```

Comment:
In other places, I think you assumed that the implementation of MCP was actually in a separate function -- which I think was a good idea. This implementation doesn't seem to match with that, though.

Reply:
Yes, complete MCP steps will be in one function. Updated the code as mentioned.